src/components/Calculator.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+export default class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="skeleton">
+        <div className="screen">
+          0
```

Comment:
- [OPTIONAL] It will be nice to align the zero digit to the right as in the [design](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md), it a good practice as a frontend dev to stick to the design as much as possible.